README.md

```diff
@@ -17,18 +11,118 @@ When run with `--send-email` it also emails this table to all the listed maintai
 
 pip install git+https://github.com/ansible-network/ansible-network-triager.git@master
 
+git clone https://github.com/ansible-network/ansible-network-triager.git
+
+cd ansible-network-triager
+
+pip install -r requirements.txt
+
+## Configuration
+
+1. add your details such as organization_name, workflow_name in `config.yaml` file:
+
+
+organization_name: "Ansible Networking"
+workflow_name: "tests.yml"
+repo_config: "repo_config.json"
+timedelta: 14
+
+
+2. update the `repo_config.json` file with your repos:
+
+
+{
+  "ansible-collections": {
+    "ci_and_bug_repos": [
+      "cisco.nxos",
+      "cisco.ios"
+    ],
+    "bug_specific_repos": [
+      "ansible.scm"
+    ]
+  }
+}
+
+
+3. Create a `.env` file in the root of your directory and add these details:
+
+
+EMAIL_SENDER=your_email@example.com
+EMAIL_PASSWORD=your_email_password
+MAINTAINERS=[{"name": "Your Name", "email": "your_email@example.com"}]
+#GITHUB_TOKEN=your_github_token (optional - enter your github token here to make authenticated requests or comment this line to make unauthenticated API requests)
+
+
 ## Usage
+
+### Local Usage
+
+1. Run the bug scrub:
+
+   triager --bugs -c config.yaml --log --send-email
+
+
+2. Run the CI report:
+
+   triager --ci -c config.yaml --log --send-email
+
+
+### Docker Usage
+
+1. Build the Docker image:
+
+   docker-compose build
+
+
+2. Run the bug scrub:
+
+   docker-compose run triager --bugs -c config.yaml --log --send-email
+
+
+3. Run the CI report:
+
+   docker-compose run triager --ci -c config.yaml --log --send-email
+
+
+### GitHub Actions
+
+1. Store your secrets (EMAIL_SENDER, EMAIL_PASSWORD, MAINTAINERS, GITHUB_TOKEN) in GitHub Actions secrets.
```

Comment:
https://github.com/ansible/ansible-content-actions take the Readme as an example to show the usage of GHA

Reply:
Done, updated the Readme section